client/my-sites/posts/post-share.jsx

```diff
@@ -212,6 +226,7 @@ export default connect(
 		const userId = getCurrentUserId( state );
 
 		return {
+			planHasRepublicizeFeature: hasFeature( state, siteId, FEATURE_REPUBLICIZE ),
```

Comment:
Do we really want to start using client side only checks? The features list controls the UI on the plans page. What happens when we decide to show only a few items?

cc @mtias @lamosty @ockham